charmcraft/templates/init/src/charm.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+# Copyright (c) {{ year }} {{ author }}
+# See LICENSE file for licensing details.
+
+import logging
+
+from ops.charm import CharmBase
+from ops.main import main
+from ops.framework import StoredState
+
+logger = logging.getLogger(__name__)
+
+
+class {{ class_name }}(CharmBase):
```

Comment:
Should we default to including a CharmEvents and a no-op event so that people know what that should look like?

Reply:
discussed in the standup: no; custom events are an advanced feature really only people writing components should be looking at